### ISSUE TITLE
Add Accessibility guidelines to vf.io

### ISSUE THREAD

**Issue by yaili** [canonical-websites/vanillaframework.io#6]
_**@barrymcgee commented on 27 Jul 2016**_

Accessibility guidelines should be added to a top level page within the docs site.

Related: I found this resource which may be informative: http://accessibility.voxmedia.com/

_Original: https://github.com/vanilla-framework/vanilla-framework/issues/413_

**Comment by barrymcgee** [comment]
Duplicate of #20
